openapi.yaml

```diff
@@ -1,279 +1,662 @@
+                - corvallis
+                - extension
+                - cascades
+                - hmsc
+                - other
+        - in: query
+          name: filter[type]
+          description: "Type of location. Valid options are: building, dining, cultural-center, parking, and other."
+          schema:
+            type: array
+            items:
+              type: string
+              enum:
+                - building
+                - dining
+                - cultural-center
+                - parking
+                - other
+        - in: query
+          name: filter[giRestroom]
+          description: >
+            If true, locations without gender inclusive restrooms will be filtered out.
+          required: false
+          schema:
+            type: boolean
+        - in: query
+          name: filter[parkingZoneGroup][oneOf]
+          description: >
+            Locations in results should match the parkingZoneGroup
+            in this query parameter.
+          required: false
+          schema:
+            type: array
+            items:
+              type: string
+              # what are the possible values for this?
+        - in: query
+          name: filter[adaParkingSpaceCount][gte]
+          description: "Only include locations with ADA parking space greater than or equal to this amount"
+          required: false
+          schema:
+            type: integer
+        - in: query
+          name: filter[motorcycleParkingSpaceCount][gte]
+          description: "Only include locations with motorcycle parking space greater than or equal to this amount"
+          required: false
+          schema:
+            type: integer
+        - in: query
+          name: filter[evParkingSpaceCount][gte]
+          description: "Only include locations with electric vehicle parking space greater than or equal to this amount"
+          required: false
+          schema:
+            type: integer
+        - in: query
+          name: filter[abbreviation]
+          description: "Search by building abbreviation"
+          required: false
+          schema:
+            type: string
+        - in: query
+          name: page[size]
+          description: "Number of results to return. Used in pagination."
+          required: false
+          schema:
+            type: integer
+            maximum: 10000
+            default: 10
+        - in: query
+          name: page[number]
+          description: "Page number of results. Used to paginate through results."
+          required: false
+          schema:
+            type: integer
+        - $ref: '#/components/parameters/pretty'
+        - in: query
+          name: include
+          required: false
+          schema:
+            type: array
+            default: []
+            items:
+              type: string
+              enum: [services]
       responses:
-        '200':
-          description: Successful response
+        "200":
```

Comment:
Should be single quotes: https://github.com/osu-mist/style-guides/blob/master/OpenAPI.md#quoting-conventions